graylog2-web-interface/src/hooks/useSearchVersionCheck.ts

```diff
@@ -37,7 +37,7 @@ export const fetchSearchVersionCheck = async ({ queryKey }) => {
 
     return data;
   } catch (e) {
-    return UserNotification.error('Could not fetch override data.');
+    return UserNotification.error(`Could not fetch override data. Error: ${e}`);
```

Comment:
Should we do something like this instead?

```suggestion
    const data = await defaultOnError(fetch(
      'GET',
      qualifyUrl(ApiRoutes.SystemSearchVersionApiController.satisfiesVersion(distribution, version).url),
    ), 'Could not fetch override data');
```